testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/log/InvalidTransactionAttributeTestCase.java

```diff
@@ -0,0 +1,88 @@
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.jboss.as.test.integration.ejb.log;
+
+import org.jboss.arquillian.container.test.api.Deployer;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.Assert.fail;
+
+
+/**
+ * Tests if the WARN message was logged when default transaction attribute was used on SFSB lifecyle method
+ * Test for [ WFLY-9509 ].
+ *
+ * @author Daniel Cihak
+ */
+@RunWith(Arquillian.class)
+public class InvalidTransactionAttributeTestCase {
+
+    private static final String ARCHIVE_NAME = "InvalidTransactionAttribute";
+
+    @ArquillianResource
+    Deployer deployer;
+
+    @Deployment(name = "invalidtransactionattribute", testable = false, managed = false)
+    public static Archive<?> deploy() {
+        JavaArchive jar = ShrinkWrap.create(JavaArchive.class, ARCHIVE_NAME + ".jar");
+        jar.addClasses(InvalidTransactionAttributeTestCase.class, StatefulBean.class, StatefulInterface.class);
+        jar.addAsManifestResource(InvalidTransactionAttributeTestCase.class.getPackage(), "ejb-jar.xml", "ejb-jar.xml");
+        return jar;
+    }
+
+    @Test
+    public void testInvalidTransactionAttributeWarnLogged() {
+        PrintStream oldOut = null;
+        String output = null;
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        try {
+            oldOut = System.out;
+            System.setOut(new PrintStream(baos));
+        } catch (Throwable t) {
+            fail(t.getMessage());
+        } finally {
+            deployer.deploy("invalidtransactionattribute");
+            System.setOut(oldOut);
+        }
+
+        try {
+            output = new String(baos.toByteArray());
+        } catch (Throwable t) {
+            fail(t.getMessage());
+        } finally {
+            Assert.assertFalse(output, output.contains("WFLYEJB0463"));
+            Assert.assertFalse(output, output.contains("ERROR"));
```

Comment:
Does the deployment result in a WARN logging now? If not there also should be 
```
Assert.assertFalse(output, output.contains("WARN")); 
```
If it does produce a WARN perhaps there's some other bug to fix.

Reply:
The deployment in upstream produces warn messages `WARN  [org.jboss.weld.Bootstrap] (MSC service thread 1-1) WELD-000146: BeforeBeanDiscovery.addAnnotatedType(AnnotatedType<?>) used for class com.arjuna.ats.jta.cdi.transactional.TransactionalInterceptorMandatory is deprecated from CDI 1.1!` for 11 different classes. Will we manage those in separate tickets?